src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -28,6 +28,14 @@ public class Engine : IPowerFxEngine
         /// </summary>
         public PowerFxConfig Config { get; }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Engine"/> class.
+        /// </summary>
+        public Engine()
+            : this(new PowerFxConfig())
+        {
+        }
```

Comment:
Couldn't we change the new constructor with

public Engine(PowerFxConfig powerFxConfig = null)
{
    Config = powerFxConfig ?? new PowerFxConfig();
}

Reply:
It's a breaking change. 
I want to be able to find-all-ref to see which ctor was being used, ie, make sure this is a compile-time decision and not a runtime. 
Likewise, it's an error to pass a null config.